Marlin/src/inc/SanityCheck.h

```diff
@@ -1234,6 +1234,15 @@ static_assert(X_MAX_LENGTH >= X_BED_SIZE && Y_MAX_LENGTH >= Y_BED_SIZE,
   #endif
 #endif
 
+/**
+ * Test case light not using the same pin as the fan
+ */
+#if ENABLED(CASE_LIGHT_ENABLE)
+  #if CASE_LIGHT_PIN == FAN_PIN
+    #error "You cannot set CASE_LIGHT_PIN equal to FAN_PIN."
+  #endif
+#endif
```

Comment:
There are a lot of pins that would be bad to coincide with `CASE_LIGHT_PIN`. Why single this one out?

Reply:
It just occurred to me for this printer, because it only has one single "controllable" pin which is used for the LED. all fans are just connected to power and run when the printer is on.
I thought it might save someone from misconfiguring